### PR TITLE
Show using wsl in windows installation docs

### LINKED ISSUE #2748: Show using WSL in Windows installation odcs
In Windows installation docs, include an information about WSL. The guide should clearly indicate that when using WSL, Linux/MacOS installation instructions should be followed e.g. rust should be installed using rustup, not through an .exe.

### PR DESCRIPTION
**Pull request by kkawula**
Closes #2748

## Introduced changes

-

## Checklist

- [ ] Linked relevant issue
- [ ] Updated relevant documentation
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/getting-started/installation.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ In this section, we will walk through the process of installing Starknet Foundry
 * [Installation](#installation)
   * [Contents](#contents)
   * [Requirements](#requirements)
-  * [Linux and MacOS](#linux-and-macos)
+  * [Linux and macOS](#linux-and-macos)
```
Comment (kkawula) [inline docs/src/getting-started/installation.md:12]: https://en.wikipedia.org/wiki/MacOS

**Review by franciszekjob — APPROVED** [review]

File: docs/src/getting-started/installation.md (hunk 1 above)
Comment (franciszekjob) [reply docs/src/getting-started/installation.md:12]: 🍎

Comment (cptartur) [reply docs/src/getting-started/installation.md:12]: I was unsure if this should be capitalized or not. But this formatting works for me

File: docs/src/getting-started/installation.md [hunk 2]
```diff
@@ -143,6 +143,10 @@ sncast --version
 
 ## Windows
 
+> 🐧 **Info** - WSL (Windows Subsystem for Linux)
+> 
+> If you're using WSL, follow the [Linux and macOS](#linux-and-macos) instructions.
```
Comment (cptartur) [inline docs/src/getting-started/installation.md:148]: Let's link this https://learn.microsoft.com/en-us/windows/wsl/install as well

And say something along the lines "you can install on windows natively but can also use [wsl]. If you are usign wsl follow this ...".

File: docs/src/getting-started/installation.md [hunk 3]
```diff
@@ -143,6 +143,13 @@ sncast --version
 
 ## Windows
 
+> 🐧 **Info** - WSL (Windows Subsystem for Linux)
+> 
+> Starknet Foundry can be installed natively on Windows, but to achieve better experience it is recommended to use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
```
Comment (cptartur) [inline docs/src/getting-started/installation.md:148]:
```suggestion
> Starknet Foundry can be installed natively on Windows, but currently, for smoother experience, it is recommended to use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
```

**Review by cptartur — APPROVED** [review]

**Pull request merged**